libraries/botbuilder-ai-orchestrator/src/orchestratorRecognizer.ts

```diff
@@ -337,100 +345,137 @@ export class OrchestratorRecognizer extends AdaptiveRecognizer implements Orches
-                    type: entityType,
-                    score: result.score,
-                    text: entityText,
-                    start: span.offset,
-                    end: span.offset + span.length,
-                });
+        // As this method is TryScoreEntities, so it's best effort only, there should
+        // not be any exception thrown out of this method.
+        try {
+            const results = await this._resolver.score(text, LabelType.Entity);
+            if (!results) {
+                throw new Error(`Failed scoring entities for: ${text}`);
+            }
 
-                // get/create $instance
-                recognizerResult.entities['$instance'] ??= {};
-                const instanceRoot = recognizerResult.entities['$instance'];
-
-                // add instanceData
-                instanceRoot[entityType] ??= [];
-                const instanceData = instanceRoot[entityType];
-                instanceData.push({
-                    startIndex: span.offset,
-                    endIndex: span.offset + span.length,
-                    score: result.score,
-                    text: entityText,
-                    type: entityType,
+            // Add full entity recognition result as a 'entityResult' property
+            recognizerResult[this.entityProperty] = results;
+            if (results.length) {
+                recognizerResult.entities ??= {};
+
+                results.forEach((result: Result) => {
+                    const entityType = result.label.name;
+
+                    // add value
+                    const values = recognizerResult.entities[entityType] ?? [];
+                    recognizerResult.entities[entityType] = values;
+
+                    const span = result.label.span;
+                    const entityText = text.substr(span.offset, span.length);
+                    values.push({
+                        type: entityType,
+                        score: result.score,
+                        text: entityText,
+                        start: span.offset,
+                        end: span.offset + span.length,
+                    });
+
+                    // get/create $instance
+                    recognizerResult.entities['$instance'] ??= {};
+                    const instanceRoot = recognizerResult.entities['$instance'];
+
+                    // add instanceData
+                    instanceRoot[entityType] ??= [];
+                    const instanceData = instanceRoot[entityType];
+                    instanceData.push({
+                        startIndex: span.offset,
+                        endIndex: span.offset + span.length,
+                        score: result.score,
+                        text: entityText,
+                        type: entityType,
+                    });
                 });
-            });
+            }
+            // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        } catch (error) {
```

Comment:
Let's remove the try catch so we don't mask some issues that should be surfaced.   It was named that way because there might not be entities extracted.   If you added try catch in c#, then leave it, however, take out the error since it's not used.  try {} catch {}

Reply:
Will remove (error)